### PR TITLE
Tried to fix whitespase (#51)

### PR DESCRIPTION
**Pull request by Drivez**

### CODE REVIEW

**Comment by Drivez** [comment]
adjusted to one linebreak

File: users.md [hunk 1]
```diff
@@ -119,35 +118,35 @@
   Facebook: 
   
 - Name: Bervianto Leo Pratama
-  Place: Bandung, Indonesia
-  Bio: Student, Institut Teknologi Bandung
-  Github: [berviantoleo](https://github.com/berviantoleo)
-  Facebook: [Bervianto Leo Pratama](https://facebook.com/bervianto.leo)
+  Place: Bandung, Indonesia
+  Bio: Student, Institut Teknologi 
+  BandungGithub: [berviantoleo](https://github.com/berviantoleo)
```
Comment (Anupam-dagar) [inline users.md]: fix this, name is coming before github.

**Comment by Drivez** [comment]
@Anupam-dagar hopefully this is fixed now

**Comment by Anupam-dagar** [comment]
star the repo and I'll merge.

**Comment by Anupam-dagar** [comment]
star the repo and I'll merge.

**Pull request merged**